Tools/AnnotationTool/UserInterface/ColorPickerUI.cs

```diff
@@ -160,10 +160,18 @@ Color GetColorForCurrentPosition()
 		var rect = m_ColorPicker.rectTransform.rect;
 
 		var dir = m_PickerTargetPosition;
-		var x = (dir.x + rect.width / 2f) / rect.width;
-		var y = (dir.y + rect.height / 2f) / rect.height;
-		var textureX = (int)(x * m_ColorPickerTexture.width);
-		var textureY = (int)(y * m_ColorPickerTexture.height);
+		dir.z = 0;
+
+		// We want to avoid the edge of the image, which has feathered pixels
```

Comment:
I would prefer a separate 'mask' texture rather than scripting in a range solution

Reply:
The fix here was just to add some padding and use the outer rect as our "canvas." That way we're essentially scaling up the texture slightly. The colors will be offset a tiny bit but there's no need for a new mask component.